modules/nf-core/glimpse/phase/meta.yml

```diff
@@ -0,0 +1,82 @@
+name: "glimpse_phase"
+description: main GLIMPSE algorithm, performs phasing and imputation refining genotype likelihoods
+keywords:
+  - phase
+  - imputation
+tools:
+  - "glimpse":
+      description: "GLIMPSE is a phasing and imputation method for large-scale low-coverage sequencing studies."
+      homepage: "https://odelaneau.github.io/GLIMPSE"
+      documentation: "https://odelaneau.github.io/GLIMPSE/commands.html"
+      tool_dev_url: "https://github.com/odelaneau/GLIMPSE"
+      doi: "10.1038/s41588-020-00756-0"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - input:
+      type: file
+      description: Input VCF/BCF file containing genotype likelihoods
+      pattern: "*.{vcf,bcf}"
+
+  - reference:
+      type: file
+      description: Reference panel of haplotypes in VCF/BCF format.
+      pattern: "*.{vcf,bcf}"
+
+  - region-file:
+      type: file
+      description: |
+        Tab delimited output txt file containing buffer and imputation regions.
+        Can be generated directly with the reference panel vcf and GLIMPSE_chunk
+      pattern: "*.txt"
+
+  - input-region:
+      type: string
+      description: Target region used for imputation, including left and right buffers (e.g. chr20:1000000-2000000).
+      pattern: "chrXX:leftBufferPosition-rightBufferPosition"
+
+  - output-region:
+      type: string
+      description: Target imputed region, excluding left and right buffers (e.g. chr20:1000000-2000000).
+      pattern: "chrXX:leftBufferPosition-rightBufferPosition"
+
+  - map:
+      type: file
+      description: File containing the genetic map.
+      pattern: "*.gmap"
+
+  - samples-file:
+      type: file
+      description: |
+        File with sample names and ploidy information.
+        One sample per line with a mandatory second column indicating ploidy (1 or 2).
+        Sample names that are not present are assumed to have ploidy 2 (diploids).
+        GLIMPSE does NOT handle the use of sex (M/F) instead of ploidy.
+      pattern:
+
+output:
```

Comment:
```suggestion
input:
  - meta:
      type: map
      description: |
        Groovy Map containing sample information
        e.g. [ id:'test', single_end:false ]
  - input:
      type: file
      description: Input VCF/BCF file containing genotype likelihoods
      pattern: "*.{vcf,bcf}"
  - reference:
      type: file
      description: Reference panel of haplotypes in VCF/BCF format.
      pattern: "*.{vcf,bcf}"
  - region-file:
      type: file
      description: |
        Tab delimited output txt file containing buffer and imputation regions.
        Can be generated directly with the reference panel vcf and GLIMPSE_chunk
      pattern: "*.txt"
  - map:
      type: file
      description: File containing the genetic map.
      pattern: "*.gmap"
  - samples-file:
      type: file
      description: |
        File with sample names and ploidy information.
        One sample per line with a mandatory second column indicating ploidy (1 or 2).
        Sample names that are not present are assumed to have ploidy 2 (diploids).
        GLIMPSE does NOT handle the use of sex (M/F) instead of ploidy.
output:
```

Please also add the index for the input and reference here :)